lib/suspenders/generators/app_generator.rb

```diff
@@ -165,18 +166,22 @@ def setup_git
       end
     end
 
+    def create_local_heroku_setup
+      say "Creating local Heroku setup"
+      build :provide_review_apps_setup_script
+      build :create_heroku_pipelines_config_file
+      build :provide_deploy_script
```

Comment:
While we are on this, should we make these names more homogeneous? They all create files, but two start with `provide` and one with `create`.